docs/framework/network-programming/how-to-enable-a-webrequest-to-use-a-proxy-to-communicate-with-the-internet.md

```diff
@@ -24,7 +24,7 @@ GlobalProxySelection.Select = proxyObject
 ## Compiling the Code  
  This example requires:  
   
--   References to the **System.Net** namespace.  
+-   A [`using` directive](~/docs/csharp/language-reference/keywords/using-directive.md) for the **System.Net** namespace.  
```

Comment:
Why not relative paths as used later in the article?
(Disclaimer: I don't know how the paths work ... just curious)

Reply:
I prefer the true relative links as well. We do use the ~ notation, which is the path to the root, when we're making global changes. But I prefer to avoid them so they work in GitHub preview too.

Reply:
Do I have to change to this?

>(../../docs/csharp/language-reference/keywords/using-directive.md)

In https://github.com/dotnet/docs/pull/6911 you'll accepted with ~. That's why I put it.

Reply:
Yep, that would be the relative path. As I said, I just prefer to use the .. notation instead of ~ for them to work in GitHub preview as well. But both are fine. If you're ok making the change, that's alright. If not, we can merge as is.

Reply:
It's not a problem at all. 😄 
I'll do it and I'll follow this to the next articles.